jujugui/static/gui/src/app/app.js

```diff
@@ -1656,6 +1626,50 @@ YUI.add('juju-gui', function(Y) {
       }
     },
 
+    createSocketURL: function(apiServer, apiPort, uuid) {
```

Comment:
Why not just: `function(host, port, uuid)`?
The server name is a bit confusing.